src/platform_impl/macos/app_delegate.rs

```diff
@@ -79,3 +107,43 @@ extern "C" fn did_resign_active(this: &Object, _: Sel, _: id) {
     }
     trace!("Completed `applicationDidResignActive`");
 }
+
+extern "C" fn application_open_file(_this: &Object, _: Sel, _sender: id, filename: id) -> BOOL {
+    trace!("Triggered `application:openFile:`");
```

Comment:
Thanks for carrying on my proud legacy of having stuffed this module full of `trace` logging!